packages/design-system/src/components/Tooltip/Tooltip.js

```diff
@@ -9,6 +9,7 @@ import { DIRECTION, ALIGMENT } from '../../constants/tooltip';
 
 class Tooltip extends React.Component {
   constructor(props) {
+    console.warn('[Design System] deprecated component - Tooltip, please use CSSTooltip or PopperTooltip')
```

Comment:
Can we ensure that it wouldn't land in the production build?

Reply:
Yup, please add check for `process.env`

Reply:
move the console.warn and check to helper - `noticeAboutDeprecation(message: string)`

Reply:
done